wgpu-core/src/command/ray_tracing.rs

```diff
@@ -1204,13 +1196,17 @@ fn iter_buffers<'a, 'b>(
             first_vertex: mesh.first_vertex,
             vertex_count: mesh.size.vertex_count,
             vertex_stride: mesh.vertex_stride,
-            indices: index_buffer.map(|index_buffer| hal::AccelerationStructureTriangleIndices::<
-                dyn hal::DynBuffer,
-            > {
-                format: mesh.size.index_format.unwrap(),
-                buffer: Some(index_buffer.as_ref()),
-                offset: mesh.index_buffer_offset.unwrap() as u32,
-                count: mesh.size.index_count.unwrap(),
+            indices: index_buffer.map(|index_buffer| {
+                let index_stride = match mesh.size.index_format.unwrap() {
+                    wgt::IndexFormat::Uint16 => 2,
+                    wgt::IndexFormat::Uint32 => 4,
```

Comment:
Might be worth slapping a public helper on `IndexFormat`, as this has shown up a few times and probably exists in user code.

```
fn index_byte_size()
```

or smth.